concurrency/MpscQueue.h

```diff
@@ -55,9 +56,9 @@ class MpscQueueBase
 #endif
         std::atomic<State> state;
         uint32_t size;
-        uint32_t _align32;
         uint8_t data[];
```

Comment:
Alignment should be explicit. Otherwise it's fragile.
You know that it's aligned because we had 2x32bit field in front and you are ensuring that we will store it in a 64bit aligned memory. But If you create an `Entry` on stack memory for example, it will requires only 32bit alignment, also is very sensitive to new fields that we can add to `Entry` in the future.

Making explicitly aligned to 64bits. Will automatically make the `Entry` to be aligned to 64bit and it's ensured by compiler that will be correct if you add new fields in the future

```suggestion
        alignas(64) uint8_t data[];
```